Code/Tools/ProjectManager/Source/ProjectExportController.cpp

```diff
@@ -139,7 +136,29 @@ namespace O3DE::ProjectManager
 
             success = false;
         }
+        else
+        {
+            AZ::Outcome<QString, QString> expectedOutputPathQuery = m_worker->GetExpectedOutputPath();
 
+            if (expectedOutputPathQuery.IsSuccess())
+            {
+                //Export was successful, show the expected output path
+                QMessageBox::StandardButton openOutputDir = QMessageBox::information(
+                        m_parent,
+                        tr("Project exported successfully!"),
+                        tr("Would you like to view the exported files?"),
+                        QMessageBox::No | QMessageBox::Yes);
+
+                if (openOutputDir == QMessageBox::Yes)
+                {
+                    QString expectedOutputPath = expectedOutputPathQuery.GetValue();
+                    if (!QDesktopServices::openUrl(QUrl::fromLocalFile(expectedOutputPath)))
+                    {
+                        qDebug() << "QDesktopServices::openUrl failed to open " << expectedOutputPath << "\n";
+                    }
+                }
```

Comment:
You should also log an error if you cannot get the expectedOutputPathQuery from the Error message of the outcome